dss/util/email.py

```diff
@@ -0,0 +1,76 @@
+import boto3
+from botocore.exceptions import ClientError
+
+# The character encoding for the email.
+CHARSET = "UTF-8"
```

Comment:
I think you can just hardcode this as the default everywhere, no need to parameterize it.

Reply:
please explain

Reply:
I mean that you can just pass the literal "UTF-8" as the argument instead of declaring it in a variable.